docs/phrases.md

```diff
@@ -108,13 +108,15 @@ Ex:
 * `I scroll to the bottom the page`
 * `I scroll to the top of the page`
 
+### Waiting For a Request to Resolve
+`I wait for "some/routeName" to resolve`
 
-### Waiting For AJAX to Finish
-`I wait for results to load`
+Wait for a request defined in the routes file to resolve. Replace `"some/routeName"` with a value from the key-value pairs in the router object.
 
-Waits for an AJAX request to finish.
+### Waiting For an Amount of Time
+`I wait for 3 seconds`
 
-This is currently not working as it should so I replaced with a `wait(1000)`. Feel free to submit PRs to fix
+Wait for the specified amount of seconds.
```

Comment:
this would wait for a route to resolve, not a set number of seconds. But let's add a route for a certain number of seconds